MobileWeather/app/src/main/AndroidManifest.xml

```diff
@@ -1,37 +1,32 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.ford.mobileweather"
-    android:versionCode="26"
-    android:versionName="1.7.14	" >
+    package="com.ford.mobileweather">
```

Comment:
I think we should rename this package, and thus all the code packages as well. `com.sdl.mobileweather` should work fine.

Reply:
Sure, I will update the package name to com.sdl.mobileweather.